frontend/src/app/reducers/browser.js

```diff
@@ -5,9 +5,11 @@ import { isMinFirefoxVersion } from '../lib/utils';
 const userAgent = navigator.userAgent.toLowerCase();
 const isUserAgentFirefox = userAgent.indexOf('firefox') > -1;
 const isUserAgentMinFirefox = isMinFirefoxVersion(isUserAgentFirefox, userAgent, config.minFirefoxVersion);
+const locale = (navigator.language || '').split('-')[0];
```

Comment:
What are the `locale` values meant to be, here? For example, it looks like this collapses `en-US`, `en-GB`, & `en-NZ` into just `en` and `es-MX`, `es-ES`, & `es-CO` into just `es`. Is that what we want? They're definitely different locales, despite shared languages.

Actually, just noticed discussion in #1686, will copy this to there